src/index.ts

```diff
@@ -1,14 +1,10 @@
 import express from "express";
-import cors from "cors";
```

Comment:
CORS was required for Frontend to work properly when it interacted with the backend service, we might need to test this by deploying the API and interacting with it from the Frontend

Reply:
Ah I see its moved to here now https://github.com/xplorer-io/xplorers-api/pull/4/files#diff-8a8ae07582c9d433ec8c2e5c4310ff8901e604f4965c5b90a49117ad46c47595R18